build.gradle

```diff
@@ -162,6 +162,15 @@ task pythonPreCommit() {
   dependsOn ":beam-sdks-python:check"
 }
 
+task pythonPostCommit() {
```

Comment:
I believe this should depend on a task defined in `sdks/python/build.gradle`. Reasons:
1. Consistent with the rest of this file.
2. The inner `sdks/python/build.gradle` config defines an inner scope (much like Bazel BUILD files define packages). The outer config file should not "reach into" files under the inner config's scope. I hope that makes sense.

Reply:
I agree.

I will make that change in a subsequent PR. The goal of this PR is to change the jenkins config to call a Gradle task, and then I'll work to clean up that gradle task.

Reply:
If that sounds fine to you, I will merge the change and continue with the rest of the process.

Reply:
That's fine. For my educatin, could explain why the last commit moved the shell invocation between build.gradle files?

Reply:
Because the script assumes it's called from root directory. Otherwise it breaks.